Snowflake.Data/Core/HttpUtil.cs

```diff
@@ -13,27 +13,14 @@ namespace Snowflake.Data.Core
 {
     class HttpUtil
     {
-        static private HttpClient httpClient;
-
-        static public HttpClient getHttpClient()

```

Comment:
I generally don't like this change. You changed it in such a way that every http request will create a new HttpClient class, which could use up a lot of resources. The original implementation will only create a single http client and that class is being shared between http requests. According to https://msdn.microsoft.com/en-us/library/system.net.http.httpclient(v=vs.110).aspx#Anchor_5, httpClient class is thread safe, so concurrent http requests should not be a problem. 

However, there is another bug in original implementation, which is that the initialization to http client singleton is not synchronized, which will lead to a null pointer exception due to the race condition. I will fix this issue in a separate pr.